front/docker/Dockerfile.playwright

```diff
@@ -0,0 +1,19 @@
+FROM alpine AS exec_as_builder
+
+RUN apk add build-base
+COPY front/docker/exec-as.c .
+RUN gcc -std=c99 -static -o /exec-as exec-as.c
+
+FROM mcr.microsoft.com/playwright
+
+COPY --from=exec_as_builder /exec-as /
+WORKDIR /app
+COPY front front
+COPY tests tests
+WORKDIR /app/front
+RUN yarn install
+RUN yarn playwright install --with-deps
+
+# Start the app
+COPY front/docker/dev-entrypoint.sh /
+ENTRYPOINT ["/dev-entrypoint.sh"]
```

Comment:
Don't use this entrypoint, it's only useful if you bindmount the whole `front/` folder.

Reply:
Done ^^